geniusyield-orderbot-framework/lib-common/GeniusYield/OrderBot/Types.hs

```diff
@@ -12,6 +12,7 @@ module GeniusYield.OrderBot.Types
     , OrderAssetPair (OAssetPair, currencyAsset, commodityAsset)
     , OrderType (..)
     , SOrderType (..)
+    , SOrderTypeI (..)
```

Comment:
I defined this class for writing `lookupBest` function inside `OrderBook.Extra` module.